enable-tracing.sh

```diff
@@ -24,18 +24,20 @@ echo "No PROJECT variable set"
 exit
 fi
 
+# FIXME
+
 array=(
-  'cart/cartservice'
-  'checkout/checkoutservice'
-  'ad/adservice'
-  'currency/currencyservice'
-  'email/emailservice'
-  'frontend/frontend'
-  'loadgenerator/loadgenerator'
-  'recommendation/recommendationservice'
-  'product-catalog/productcatalogservice'
-  'payment/paymentservice'
-  'shipping/shippingservice'
+  'onlineboutique/cartservice'
+  'onlineboutique/checkoutservice'
+  'onlineboutique/adservice'
+  'onlineboutique/currencyservice'
+  'onlineboutique/emailservice'
+  'onlineboutique/frontend'
+  'onlineboutique/loadgenerator'
+  'onlineboutique/recommendationservice'
+  'onlineboutique/productcatalogservice'
+  'onlineboutique/paymentservice'
+  'onlineboutique/shippingservice'
   'orders/orderservice'
```

Comment:
Change 'orders/orderservice' to 'onlineboutique/orderservice'

Reply:
Done.